### PR TITLE
Persian/Farsi Translation of SemVer v2.0.0

### PR DESCRIPTION
**Pull request by mhadaily**
Hi, 
I have translated SemVer to Persian/Farsi. I have tried to edit language to be the best fit for Persian speakers. 

Thanks, 
Majid

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ _site
 *.swp
 .DS_Store
 Gemfile.lock
+semver.org.iml
+.idea
```
Comment (haacked) [inline .gitignore]: What are these here for?

**Review by haacked — COMMENTED** [review]
Thank you so much! I cannot read Farsi so would you be able to find someone who can proofread your translation to help review it?

File: .gitignore (hunk 1 above)
Comment (mhadaily) [reply .gitignore]: these have been added to .gitigone automatically by my IDEA. I can remove and push again if you like,

**Comment by mhadaily** [comment]
Yes, Sure. I'll get someone to review my translation.

**Comment by farzadghanei** [comment]
Hi
I reviewed the translation and the document, and it seems a good translation.
Thanks

**Comment by haacked** [comment]
Thanks everyone! :sparkles:

**Pull request merged**